content/pt-br/docs/concepts/architecture/_index.md

```diff
@@ -1,5 +1,244 @@
+	class="diagram-large" 
+>}}
+
+## Componentes da camada de gerenciamento
+
+Os componentes da camada de gerenciamento tomam decisões globais sobre o cluster (por exemplo, alocação),
+bem como detectam e respondem a eventos do cluster (por exemplo, iniciar um novo pod quando
+o campo {{< glossary_tooltip text="replicas" term_id="replica" >}} de um {{< glossary_tooltip text="Deployment" term_id="deployment" >}}
+não está satisfeito)
+
+Os componentes da camada de gerenciamento podem ser executados em qualquer máquina no cluster.
+Contudo, para simplificar, scripts de configuração normalmente iniciam todos os componentes
+da camada de gerenciamento na mesma máquina, e não executam contêineres de usuários nesta máquina.
+Confira [Criando clusters de Alta Disponibilidade com kubeadm (em inglês)](/docs/setup/production-environment/tools/kubeadm/high-availability/)
+para um exemplo de uma configuração da camada de gerenciamento que é executada entre múltiplas máquinas.
+
+### kube-apiserver
+
+{{< glossary_definition term_id="kube-apiserver" length="all" >}}
+
+### etcd
+
+{{< glossary_definition term_id="etcd" length="all" >}}
+
+### kube-scheduler
+
+{{< glossary_definition term_id="kube-scheduler" length="all" >}}
+
+### kube-controller-manager
+
+{{< glossary_definition term_id="kube-controller-manager" length="all" >}}
+
+Existem diversos tipos diferentes de controladores. Alguns exemplos deles são:
+
+- Controlador de Nós: responsável por perceber e responder quando nós caem.
+- Controlador de {{< glossary_tooltip text="Job" term_id="job" >}}: Observa os objetos do
+tipo Job que representam tarefas pontuais, então cria Pods para executar estas tarefas até
+a conclusão.
+- Controlador de {{< glossary_tooltip text="EndpointSlice" term_id="endpoint-slice" >}}: Popula
+objetos do tipo EndpointSlice (para fornecer um vínculo entre Serviços e Pods).
+- Controlador de {{< glossary_tooltip text="ServiceAccount" term_id="service-account" >}}:
+Cria contas de serviço padrão para novos {{< glossary_tooltip text="namespaces" term_id="namespace" >}}.
+
+A lista acima não é completa.
+
+### Gerenciador de controlador de nuvem
+
+{{< glossary_definition term_id="cloud-controller-manager" length="short" >}}
+
+O gerenciador de controlador de nuvem apenas executa controladores que são específicos para
+o seu provedores de nuvem. Se você está executando Kubernetes nas suas próprias instalações
+(_on premises_), ou em um ambiente de aprendizado no seu PC próprio, o cluster não tem um
+gerenciador de controlador de nuvem.
+
+Assim como com o kube-controller-manager, o gerenciador de controlador de nuvem combina vários 
+circuitos de controles lógicos independentes em um único binário que você executa como um
+único processo.
+Você pode escalar horizontalmente (executar mais de uma cópia) para melhorar a performance
+ou ajudar a tolerar falhas.
```

Comment:
(I assume that the existing wording from https://kubernetes.io/pt-br/docs/concepts/overview/components/ is OK to keep)
```suggestion
Você pode escalonar horizontalmente (executar mais de uma cópia) para melhorar
o desempenho ou para auxiliar na tolerância a falhas.
```